### ISSUE TITLE
The Outline client v 1.10.1.0 totaly not works on windows 10 22H2 build 19045.2846

### ISSUE THREAD

**Issue by Alexandr2021i** [Jigsaw-Code/outline-client#1633]
**Perform clear install on clear Windows 10 setup, results:**

Outline is starting
loading web app from file:///c:/Program%20Files%20(x86)/Outline/resources/app.asar/www/index_electron.html?
debug=true&appName=Outline
Could not load active tunnel:  [Error: ENOENT: no such file or directory, open 'C:\Users\Alexandr\AppData\Roaming\Outline\connection_store'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Users\\Alexandr\\AppData\\Roaming\\Outline\\connection_store'
}
connecting to 4e48d09e-ef66-4e4d-bf0a-2aadca74882a...
using tun2socks to check connectivity
[EXIT - tun2socks.exe]: Exited with code 0
UDP support: true
starting routing daemon
Routing daemon socket setup failed Error: connect ENOENT \\.\pipe\OutlineServicePipe
    at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'connect',
  address: '\\\\.\\pipe\\OutlineServicePipe'
}
could not connect: w (routing daemon is not running)
[EXIT - tun2socks.exe]: Killed by signal SIGTERM
tun2socks terminated due to Error: Process terminated by signal: SIGTERM
disconnected from 4e48d09e-ef66-4e4d-bf0a-2aadca74882a
installing outline routing service...
connecting to 4e48d09e-ef66-4e4d-bf0a-2aadca74882a...
using tun2socks to check connectivity
[SC] DeleteService SUCCESS
[SC] CreateService SUCCESS
The OutlineService service is starting.
The OutlineService service is not started.
More help is available by typing NET HELPMSG 3521.
The OutlineService service could not be started.
The service did not report an error.
More help is available by typing NET HELPMSG 3534.

outline routing service installed successfully

[EXIT - tun2socks.exe]: Exited with code 0
UDP support: true
starting routing daemon
Routing daemon socket setup failed Error: connect ENOENT \\.\pipe\OutlineServicePipe
    at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'connect',
  address: '\\\\.\\pipe\\OutlineServicePipe'
}
could not connect: w (routing daemon is not running)
[EXIT - tun2socks.exe]: Killed by signal SIGTERM
tun2socks terminated due to Error: Process terminated by signal: SIGTERM
disconnected from 4e48d09e-ef66-4e4d-bf0a-2aadca74882a
installing outline routing service...
[SC] DeleteService SUCCESS
[SC] CreateService SUCCESS
The OutlineService service is starting.
The OutlineService service is not started.
More help is available by typing NET HELPMSG 3521.
The OutlineService service could not be started.
The service did not report an error.
More help is available by typing NET HELPMSG 3534.

outline routing service installed successfully

**... and so on cyclically...**

In other words, the client constantly reinstalls the service, and the service does not start, while not reporting anything at all about the error.

**Comment by jyyi1** [comment]
Hi @Alexandr2021i , thanks for reporting the issue.

The log message indicates that something is preventing the `OutlineService` from starting:

```
The OutlineService service is starting.
The OutlineService service is not started.
More help is available by typing NET HELPMSG 3521.
The OutlineService service could not be started.
The service did not report an error.
More help is available by typing NET HELPMSG 3534.
```

Can you manually start the service in Task Manager or services.msc, and then retry connecting in Outline Client?

![image](https://github.com/Jigsaw-Code/outline-client/assets/93548144/97bcd688-ec5a-431f-8896-3a394cd95754)

**Comment by Alexandr2021i** [comment]
Manual starts & stop immediately. Logs does nothing.

**Comment by Alexandr2021i** [comment]
Run service exe from console - same result, no any info in console, just exit..

**Comment by jyyi1** [comment]
Hi @Alexandr2021i , could you please take a look at the Windows Event Logs to see whether there are any error reports from the service?

![image](https://github.com/Jigsaw-Code/outline-client/assets/93548144/a89cca8e-431b-4135-a1e1-82601ce87465)

Also, if you have installed any Anti-Virus software, that might block our service from running, too.

**Comment by ruslaniv** [comment]
How do I even check outline logs? There is no way in the client to display the logs!

**Comment by jyyi1** [comment]
Hi @ruslaniv , you need to launch Outline through PowerShell. "Right click start button" -> "Terminal", then run the following command

```
& 'C:\Program Files (x86)\Outline\Outline.exe'
```

You might need to change `C:\Program Files (x86)` if your system folder is different.
